src/chime_dash/app/pages/sidebar.py

```diff
@@ -196,7 +246,12 @@ def get_html(self) -> List[ComponentMeta]:
                     )
                 )
             elements.append(element)
-
+        parameters_input = dcc.Input(id='parameters_input')
+        elements.append(parameters_input)
+        store = dcc.Store(id='store')
+        elements.append(store)
+        load_button = Button("Load parameters", id='load_button')
+        elements.append(load_button)
```

Comment:
Pretty minor, but another option would be doing elements.extend(parameters_input, store, load_button) at the end.